src/randoop/NoExceptionCheck.java

```diff
@@ -3,8 +3,10 @@
 import randoop.sequence.Execution;
 
 /**
- * Checks that no exception is thrown after a given statement
- * in a sequence.
+ * Checks that an exception is not thrown by a given statement in a sequence. 
```

Comment:
"an exception is not thrown" could be interpreted as "no exception is thrown" or "some specific exception is not thrown".  Can you make the text explicit?